services/harmony/app/util/url.ts

```diff
@@ -8,9 +8,7 @@ import * as url from 'url';
  */
 function _getProtocol(req): string {
   const host = req.get('host');
-  return (host.startsWith('localhost')
-    || host.startsWith('internal-harmony') // using an internal harmony load balancer
-    || host.startsWith('127.0.0.1')) ? 'http' : 'https';
+  return (host.startsWith('localhost') || host.startsWith('127.0.0.1')) ? 'http' : 'https';
```

Comment:
do these work when Harmony is running in k8s?

Reply:
The only change to this line was removing internal-harmony so shouldn't be an issue. I verified I was able to successfully complete a request with harmony in a box after building all the images.